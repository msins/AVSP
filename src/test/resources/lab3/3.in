8 8
4 4 3 2 4 X 5 2
1 2 3 X X 1 1 3
3 1 5 X 2 5 5 2
X 4 5 4 3 5 X X
4 1 5 4 X 1 2 1
5 1 X 1 X 4 1 3
X 4 X 2 4 1 X X
X 5 1 4 3 2 4 X
2
7 8 1 4
7 8 0 4
