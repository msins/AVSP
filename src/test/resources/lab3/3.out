1.766
2.457
